tsconfig.json

```diff
@@ -1,5 +1,11 @@
 {
   "extends": "./packages/build/config/tsconfig.common.json",
   "include": ["packages"],
-  "exclude": ["node_modules/**", "packages/*/node_modules/**", "**/*.d.ts"]
+  "exclude": [
+    "node_modules/**",
+    "packages/*/node_modules/**",
+    "**/*.d.ts",
+    "packages/*/test/sandbox/**",
```

Comment:
I thought we were switching to `packages/*/.sandbox` for the recommended TestSandbox Path within the project?

Reply:
AFAICT, this change has not happened yet.